### ISSUE TITLE
feature request/suggestion: add (recenter) after counsel-org-goto-action / imenu-default-goto-function

### ISSUE THREAD

**Issue by dalanicolai** [abo-abo/swiper#2838]
Currently, when using counsel-org-goto, and using `C-M-n` `ivy-next-line-and-call`, it is unhandy that the cursor/headline is placed at the bottom of the screen because then the "content" of the headline is not shown on the screen (it is "below" the screen).

Therefore, I currently have in my dotfile:
```
  (advice-add 'counsel-org-goto-action :after #'recenter)

  (with-eval-after-load 'imenu
    (add-function :after imenu-default-goto-function #'recenter))
```
But I would suggest you make this the default (and probably also add a `org-goto-default-goto-function` customizable variable similar to the `imenu-default-goto-function` variable.

Let me know if you would like me to create a PR for this.

**Comment by basil-conto** [comment]
Makes sense, and there is already some precedent: see the user option `swiper-action-recenter` and how it's used, as well the various bindings for `ivy-call-and-recenter` and `swiper-recenter-top-bottom`.  Perhaps it would be good to generalise this behaviour across more Ivy commands.

Thanks.
